maths/series/geometric_series.py

```diff
@@ -11,53 +10,54 @@
 """
 
 
-def geometric_series(nth_term: int, start_term_a: int, common_ratio_r: int) -> list:
-    """Pure Python implementation of Geometric Series algorithm
+def geometric_series(
+    nth_term: float, start_term_a: float, common_ratio_r: float
+) -> list[float]:
+    """
+    Pure Python implementation of Geometric Series algorithm
+
     :param nth_term: The last term (nth term of Geometric Series)
     :param start_term_a : The first term of Geometric Series
     :param common_ratio_r : The common ratio between all the terms
     :return: The Geometric Series starting from first term a and multiple of common
         ration with first term with increase in power till last term (nth term)
+
     Examples:
-    >>> geometric_series(4, 2, 2)
-    [2, '4.0', '8.0', '16.0']
     >>> geometric_series(4.0, 2.0, 2.0)
-    [2.0, '4.0', '8.0', '16.0']
+    [2.0, 4.0, 8.0, 16.0]
+    >>> geometric_series(4.0, 2.0, 2.0)
```

Comment:
It is still useful to test what happens is the caller ignores the type hints.  They _often_ will.
```suggestion
    >>> geometric_series(4, 2, 2)
```